hoot-core/src/main/cpp/hoot/core/io/OgrReader.cpp

```diff
@@ -932,11 +932,6 @@ void OgrReaderInternal::open(const QString path, QString layer)
     _pendingLayers = getLayersWithGeometry(path);
   }
   LOG_VART(_pendingLayers);
-  if (Log::getInstance().getLevel() == Log::Info)
-  {
-    std::cout << ".";
-    std::cout.flush();
-  }
```

Comment:
This just printed out a dot in the middle of the busy output.  I figured it wasn't necessary as there is other output that does the same thing.